apps/tup-cms/src/taccsite_cms/settings_custom.py

```diff
@@ -32,6 +32,15 @@
     }
 }
 
+STORAGES = {
+    "default": {
+        "BACKEND": "django.core.files.storage.FileSystemStorage",
+    },
+    "staticfiles": {
+        "BACKEND": "django.contrib.staticfiles.storage.ManifestStaticFilesStorage",
+    },
+}
```

Comment:
> [!IMPORTANT]
> To add this causes an error:
> ```log
> Post-processing 'ui/assets/generics/fonts.css' failed!
> 
> …
> 
> ValueError: The file 'ui/assets/generics/fonts/BentonSans-Regular.otf' could not be found with <django.contrib.staticfiles.storage.ManifestStaticFilesStorage object at 0x7ffffc9b5280>.
> ```
> To avoid it:
> **A. Either** [document the use of `python collectstatic … --ignore assets/*/font*.css`](https://github.com/TACC/tup-ui/pull/400/commits/463d10a) in:
> - Core-CMS
> - Core-CMS-Custom
> - Core-CMS-Resources
> <details><summary><strong>B. Or</strong> fix the root problem with the fonts.</summary>
>
> > In some of these font files `url(fonts/...)` needs to be `url(../fonts/..)` e.g. https://github.com/TACC/Core-Styles/blob/v2.22.4/dist/settings/font.css.
> > — (paraphrased) @jarosenb
>
> > The paths to fonts have — in the past — been manipulated via https://github.com/TACC/Core-Styles/blob/v2.22.4/src/.postcssrc.base.yml#L37-L43.
> > — (paraphrased) @wesleyboar 


</details>

Reply:
Resolved by documented this in TACC/tup-ui (bfa1872).

Reply:
Bug reported as [TUP-708](https://tacc-main.atlassian.net/browse/TUP-708).